README.md

```diff
@@ -1,2 +1,25 @@
-# book-library
-Manchester Codes final project for Backend module to demonstrate knowledge of all topics covered
+# Book Library API Project
+---
+## About
+---
+Final project for Backend module of Manchester Code built using Express, MySQL and Sqequelize.
+
+The app uses Chai, Mocha and Supertest for testing.
+# Install
+---
+* Run a container inside a MySQL image
+* Fork and Clone down this repo
+* Run `npm install`
+* Create a .env and .env.test file with the local variables
+*DB_NAME*
+*DB_PASSWORD*
+*DB_USER*
+*DB_HOST*
+*DB_PORT*
+*PORT*
+* Run `npm start` to hit endpoints
+* Run `npm t` to test
+# Author
```

Comment:
It would be good to have a section here explaining how to use the API, with what endpoints are provided and how they are used